sdk/src/types/block/output/mod.rs

```diff
@@ -217,6 +217,31 @@ impl Output {
         }
     }
 
+    /// Returns all the mana held by the output, which is potential + stored, all decayed.
+    pub fn all_mana(
```

Comment:
What's a good name for this?

Reply:
Maybe available_mana(), total_mana(), input_mana()

Reply:
maybe `total_mana_with_decay`

Reply:
how about renaming L209 to `stored_mana` and just call this one `mana`?

Reply:
Went for `available_mana`